bot/cogs/views/sticky.py

```diff
@@ -0,0 +1,164 @@
+import discord
+from discord import Interaction
+from discord.ext import commands
+
+from bot.config import config
+
+from datetime import datetime
+from typing import Optional, List, Any, Dict
+
+
+class StickyPagination(discord.ui.View):
+
+    def __init__(self, *,
+        timeout: Optional[float] = 180,
+        PreviousButton: discord.ui.Button = discord.ui.Button(emoji=discord.PartialEmoji(name="\U000025c0")),
+        NextButton: discord.ui.Button = discord.ui.Button(emoji=discord.PartialEmoji(name="\U000025b6")),
+        PageCounterStyle: discord.ButtonStyle = discord.ButtonStyle.grey,
+        initial_page_number: int = 0,
+        ephemeral: bool = False,
+        list_data: List[Dict[str, Any]]
+    ) -> None:
+        super().__init__(timeout=timeout)
+
+        self.PreviousButton = PreviousButton
+        self.NextButton = NextButton
+        self.PageCounterStyle = PageCounterStyle
+        self.initial_page_number = initial_page_number
+        self.ephemeral = ephemeral
+        self.list_data = list_data
+
+        self.pages: List[discord.Embed] = []
+        self.page_counter: discord.ui.Button = None
+        self.current_page = None
+        self.total_page_count = None
+        self.ctx = None
+        self.message = None
+
+    async def construct_pages(self, ctx: commands.Context, list_data: List[Dict[str, Any]]) -> None:
+        N_LIST = 20
+
+        total_data = len(list_data)
+        if total_data % N_LIST:
+            self.total_page_count = total_data//N_LIST + 1
+        else:
+            self.total_page_count = total_data//N_LIST
+
+        if self.total_page_count:
+            for page_num in range(self.total_page_count):
+                page_data_list = [
+                    list_data[(page_num * N_LIST):(page_num * N_LIST) + N_LIST//2],
+                    list_data[(page_num * N_LIST) + N_LIST // 2:(page_num + 1) * N_LIST]
+                ]
+
+                embed = discord.Embed(
+                    color=discord.Color.gold(),
+                    title='WARNET STICKY MESSAGE',
+                    description='**Sticky message yang terdapat pada server WARNET**',
+                    timestamp=datetime.now()
+                )
+
+                for sticky_data_list in page_data_list:
+                    if sticky_data_list == page_data_list[1] and len(page_data_list[1]) == 0:
+                        continue
+
+                    field_value = ''
+                    field_name = 'Channel  |  Message' if sticky_data_list == page_data_list[
+                        0] else '|'
```

Comment:
This code style seems hard to read. If you really want to make it shorter, you can change it like this:

```suggestion
                    field_name = ''
                    if sticky_data_list == page_data_list[0]:
                        field_name = 'Channel  |  Message'
                    else:
                        field_name = '|'
```